### ISSUE TITLE
samtools_name_sort arguments are out of date in some scripts

### ISSUE THREAD

**Issue by thomas-reimonn** [ENCODE-DCC/atac-seq-pipeline#289]
## **Describe the bug**
In encode_task_filter.py, line 288, the call is:
    nmsrt_bam = samtools_name_sort(bam, nth, out_dir)
and it should be
    nmsrt_bam = samtools_name_sort(bam, nth, mem_gb, out_dir)
                                                                              ^missing argument

The same argument is missing in encode_task_bam2ta.py, line 70. 
    nmsrt_bam = samtools_name_sort(bam, nth, out_dir)
                                                                            ^missing argument

## **OS/Platform**
Docker 2.4.0.0 on Ubuntu 16.04 with singularity

## **Caper configuration file**
N/A

## **Input JSON file**
N/A

**Comment by leepc12** [comment]
Did you use the latest pipeline v1.9?
I checked v1.9 docker container and it has the parameter `mem_gb` .
```
root@376956ce8d7b:/software/atac-seq-pipeline/src# cat encode_task_filter.py  | grep mem_gb
def rm_unmapped_lowq_reads_se(bam, multimapping, mapq_thresh, nth, mem_gb, out_dir):
        qname_sort_bam = samtools_name_sort(bam, nth, mem_gb, out_dir)
                res_param=get_samtools_res_param('sort', nth=nth, mem_gb=mem_gb),
                res_param=get_samtools_res_param('sort', nth=nth, mem_gb=mem_gb),
def rm_unmapped_lowq_reads_pe(bam, multimapping, mapq_thresh, nth, mem_gb, out_dir):
                res_param=get_samtools_res_param('sort', nth=nth, mem_gb=mem_gb),
            res_param=get_samtools_res_param('sort', nth=nth, mem_gb=mem_gb),
            args.nth, args.mem_gb, args.out_dir)
            args.nth, args.mem_gb, args.out_dir)
    samstat(final_bam, args.nth, args.mem_gb, args.out_dir)
```

**Comment by thomas-reimonn** [comment]
I’m on the master branch, and I also just checked 1.9. There are places mem_gb is missing. For example, in line 74
qname_sort_bam = samtools_name_sort(bam, nth, mem_gb, out_dir)
the samtools_name_sort call includes mem_gb.

But line 288
                nmsrt_bam = samtools_name_sort(bam, nth, out_dir)
does not.

Sincerely,
Thomas Reimonn
M.D./Ph.D. Student (4th year)
University of Massachusetts Medical School
Thomas.reimonn@umassmed.edu<mailto:Thomas.reimonn@umassmed.edu>
(508) 308-9408


Confidentiality Notice:
This e-mail message, including any attachments, is for the sole use of the intended recipient(s) and may contain confidential, proprietary and privileged information. Any unauthorized review, use, disclosure or distribution is prohibited. If you are not the intended recipient, please contact the sender immediately and destroy or permanently delete all copies of the original message.

**Comment by leepc12** [comment]
You are right. This will be fixed in the next release, which will be released soon.
